src/optimizers/quantizer.cpp

```diff
@@ -0,0 +1,187 @@
+#include <cmath>
+
+#include "optimizers/quantizer.h"
+#include "tensors/tensor_allocator.h"
+#include "tensors/tensor_operators.h"
+
+#include "functional/functional.h"
+
+namespace marian {
+
+/* simulate a fixed quantization for values in data.
+ * For example:
```

Comment:
please use "linear quantization" (`quantizeLinearly()`), or possibly "fixed-point quantization" if applicable (not sure)

Reply:
updated to fixedPointQuantization(..)